ReactAndroid/src/main/java/com/facebook/react/ReactActivityDelegate.java

```diff
@@ -65,7 +67,7 @@ public ReactActivityDelegate(
   }
 
   protected ReactRootView createRootView() {
-    return new ReactRootView(getContext());
+    return new ReactAndroidTVViewManager(getContext());
```

Comment:
We should find some other way instead of making changes here. I'd suggest that we:
1) Add necessary overrides to ReactRootView (seems like just adding `dispatchKeyEvent` should be enough)
2) Proxy call to this method to a new class called `ReactAndroidTVRootViewHelper` where all the TV related logic from TVViewMagaer will be put